autoload/coqtail.vim

```diff
@@ -361,7 +361,7 @@ function! coqtail#start(...) abort
       autocmd BufWinLeave <buffer> call coqtail#panels#hide()
       autocmd BufWinEnter <buffer>
         \ call coqtail#panels#open(0) | call s:call('refresh', '', {})
-      autocmd QuitPre <buffer> call coqtail#stop()
+      autocmd BufDelete <buffer> call coqtail#stop()
```

Comment:
The problem with `BufDelete` is that on `:q` if the main buffer is only in one window `BufWinLeave` is triggered first and it raises `E855: Autocommands caused command to abort`. I think  what could work instead is:
```suggestion
      autocmd QuitPre <buffer> if len(win_findbuf(expand('<abuf>'))) == 1 | call coqtail#stop() | endif
```
That way it still calls `coqtail#stop` if it's the last window but otherwise just closes normally.